core/src/main/java/com/linecorp/armeria/server/RouteBuilder.java

```diff
@@ -461,7 +465,9 @@ public String toString() {
 
     private static PathMapping getPathMapping(String pathPattern) {
         requireNonNull(pathPattern, "pathPattern");
-
+        if (pathPattern.isEmpty()) {
+            return new ExactPathMapping(pathPattern);
+        }
```

Comment:
IIUC, `ExactPathMapping` will throw an  IllegalArgumentException on an empty string.

Reply:
This code was removed in 1d30371 :)